lib/tasks/schools/daily_regeneration.rake

```diff
@@ -3,7 +3,7 @@ namespace :school do
   task daily_regeneration: :environment do
     puts "#{DateTime.now.utc} Run daily regeneration for all process data schools start"
     BenchmarkResultGenerationRun.create!
-    School.process_data.each do |school|
+    School.process_data.order(:name).limit(5).each do |school|
```

Comment:
Do you mean to limit to 5 here?

Reply:
I didn't! that was left over from manual testing. I've pushed an update.

The ordering was intentional though to make it a little more predictable when different schools get executed in the queue.